doc-Introduction_to_Self_Service_Portal/topics/My_Services_Tab.adoc

```diff
@@ -102,3 +106,14 @@ You can open the Cockpit console for a virtual machine from the image:ssui-acces
 image:cockpit-login-nb.png[Cockpit Login]
 . Once you have logged in, you will see the tabs for the dashboard and the individual machines added to Cockpit. 
 image:cockpit-ui-nb.png[Cockpit UI]
+
+[[vm-power-operations]]
+=== Power Operating the Virtual Machine
+
+Follow this procedure to perform power operations on virtual machines, that is start, stop, or suspend, depending on the current state of the virtual machine. 
```

Comment:
I would change "that is" to "such as" - I think that's the intended meaning here.

Reply:
Fixed.